.changeset/clever-elephants-act.md

```diff
@@ -0,0 +1,5 @@
+---
+'@audius/harmony': minor
```

Comment:
Marking this as minor even though it's a breaking change because we aren't at 1.0.0 yet